packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -763,6 +793,17 @@ describe('AddresslistVoting', function () {
         ).to.equal(id);
       });
 
+      it('reverts on voting None', async () => {
+        await advanceIntoVoteTime(startDate, endDate);
+
+        expect(await voting.callStatic.vote(id, VoteOption.Yes, false)).to.not
```

Comment:
and why the `callStatic`?

Reply:
Probably he wants to not mark the vote from the caller, otherwise next line would be wrong for different reason.

Is it too hard to remove callStatic and do `voting.connect(signers[1])` i guess you would need to have voting power for this user, but maybe this is better. `callStatic` might be confusing for future devs even though we understand it.

Reply:
Yes, I don't want to use up the voters vote.

I double checked and `expect (await ...` seems to be correct.
https://docs.ethers.org/v5/api/contract/contract/#contract-callStatic